solutions/18.js

```diff
@@ -0,0 +1,45 @@
+// Find missing number in array
+// Return the missing number in an input array of consecutively increasing numbers.
+// input: [5,6,8,9]
+// output: 7
+
+// Zach Nagatani
+
+/**
+ * Finds and returns the missing number from an array of consecutively increasing numbers
+ * @param {Number[]} arr - An array of consecutively increasing numbers
+ * @returns {Number|null} missingNum - the number missing from the input array or null if none is found
+ */
+// const solution = arr => {
```

Comment:
instead of commenting out previous solution, could you rename them? `solution`, `solution1`,  and export  all the solutions.